### PR TITLE
FRED2 as a conda package

### PR DESCRIPTION
**Pull request by apeltzer**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/fred/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{% set name = "fred" %}
```
Comment (bgruening) [inline recipes/fred/meta.yaml]: should that be fred2?

File: recipes/fred/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    - python -c "from Fred2.Core import Allele, Peptide, Protein,generate_peptides_from_proteins"
+
+about:
+  home: http://nf-co.re/
+  license: MIT
+  summary: 'Python package with helper tools for the nf-core community.'
+
+extra:
+  container:
+    # click requires a unicode locale when used with Python 3
```
Comment (bgruening) [inline recipes/fred/meta.yaml]: But you are not using click here isn't it?

File: recipes/fred/meta.yaml (hunk 1 above)
Comment (apeltzer) [reply recipes/fred/meta.yaml]: Hm, could be - should I rename it ?

File: recipes/fred/meta.yaml (hunk 2 above)
Comment (apeltzer) [reply recipes/fred/meta.yaml]: Whoops, thats a relict of my template. I'll remove it.

Comment (apeltzer) [reply recipes/fred/meta.yaml]: Changed it, renamed the package as well to fred2

**Pull request merged**